testing/esp/esdlcmd/esdlcmd-test.py

```diff
@@ -231,7 +336,15 @@ def parse_options():
                         help='Enable debug logging of test cases',
                         action='store_true', default=False)
 
+    parser.add_argument('-c', '--commands',
+                         help='esdl commands to run tests for, use once for each command or pass "all" to test all commands. Defaults to "all".',
+                         action="append", choices=command_values)
```

Comment:
Any reason not to use "default" parameter rather than the extra code in 345.

Reply:
That was my first approach, but I saw that using `action="append"` with a default always results in the list containing the default in addition to any values supplied on the command line. A careful reading of the append action documentation does mention it: https://docs.python.org/3.12/library/argparse.html?highlight=argparse#action